ts/sveltelib/store-subscribe.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+import type { Readable, Unsubscriber } from "svelte/store";
+
+interface StoreAccessors {
+    subscribe: () => void;
+    unsubscribe: () => void;
+}
+
+// Prevent double subscriptions / unsubscriptions
```

Comment:
How do they happen?

Reply:
I don't know the code well enough to fully follow that, but I'll take your word for it :-) Thanks for elaborating.